client/my-sites/site-settings/form-general.jsx

```diff
@@ -442,6 +444,46 @@ const FormGeneral = React.createClass( {
 		);
 	},
 
+	helloVoteOption() {
```

Comment:
Yes, I think this is ok because we mention that it is for US-based users, and it will only be active for about 1 week.  We can also add a link to the related en.blog post when it goes live.